### PR TITLE
Re-add Cranelift publish script

### PR DESCRIPTION
**Pull request by fitzgen**

### CODE REVIEW

File: scripts/publish-cranelift.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euo pipefail
+
+# This is a convenience script for maintainers publishing a new version of
+# Cranelift to crates.io. To use, first bump the version number by running the
+# `scripts/cranelift-version.sh` script, then run this script, and run the
+# commands that it prints.
+#
+# Don't forget to push a git tag for this release!
```
Comment (bnjbvr) [inline scripts/publish-cranelift.sh]: The previous script did print on the terminal the instructions to run for the `git tag` et al., is this something we'd like to keep on doing here?

File: scripts/publish-wasmtime.sh [hunk 2]
```diff
@@ -2,12 +2,19 @@
 set -euo pipefail
 
 # This is a convenience script for maintainers publishing a new version of
-# Wasmtime to crates.io. To use, bump the version number below, run the
-# script, and then run the commands that the script prints.
+# Wasmtime to crates.io. To use, first bump the Wasmtieme versions with
```
Comment (bnjbvr) [inline scripts/publish-wasmtime.sh]: nit: Wasmtime

File: scripts/publish-wasmtime.sh [hunk 3]
```diff
@@ -2,12 +2,19 @@
...
+# `scripts/bump-wasmtime-version.sh` and Cranelifte versions with
```
Comment (bnjbvr) [inline scripts/publish-wasmtime.sh]: nit: Cranelift

File: scripts/publish-wasmtime.sh [hunk 4]
```diff
@@ -2,12 +2,19 @@
...
+# `scripts/bump-cranelift-version.sh`, then run this script, and run the
+# commands that it prints.
+#
+# Don't forget to push a git tag for this release!
```
Comment (bnjbvr) [inline scripts/publish-wasmtime.sh]: May I suggest to actually print this message? It's likely users of this script won't look at its source when using it.

**Comment by fitzgen** [comment]
Good eye on the typos! I also made it so that the scripts echo how to tag the release.

> Do we have something that will bump the version numbers automatically in Cranelift's Cargo.toml files?

[Yep!](https://github.com/bytecodealliance/wasmtime/blob/8598295bc4ad3c820b8a55dfd642ee2e8d8a652b/scripts/cranelift-version.sh)

**Pull request merged**